src/screens/Landing/index.js

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 import InfoSection from './InfoSection';
 import Subscribe from './Subscribe';
 import Footer from '../../components/common/Footer';
+import Navbar from '../../components/Navbar/Navbar/Navbar'
```

Comment:
We're going to need to fix this nesting. Put the navbar folder where the footer is. Similar to footer it should only be one file where everything lives.  components/common/Navbar

Reply:
It wouldn't let me assign common/Navbar because footer already has that file.  I went with commonNB.  Please let me know if that is okay, Thanks!